show-vars.md

```diff
@@ -0,0 +1,111 @@
+---
+title: SHOW (session settings)
+summary: The SHOW statement displays the current settings for the client session.
+toc: false
+---
+
+The `SHOW` [statement](sql-statements.html) can display the value of either one or all of
+the session setting variables.
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to display the session settings.
+
+## Synopsis
+
+General syntax:
+
+~~~
+   SHOW <variable name>
+   SHOW ALL
+~~~
+
+The variable name is case insensitive.
+It may be enclosed in double quotes; this is useful if the variable name itself contains spaces.
+
+Special syntax cases supported for compatibility:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SHOW TRANSACTION PRIORITY` | `SHOW "transaction priority" |
+| `SHOW TRANSACTION ISOLATION LEVEL` | `SHOW "transaction isolation level" |
+| `SHOW TIME ZONE` | `SHOW "time zone" |
+| `SHOW TRANSACTION STATUS` | `SHOW "transaction status" |
+
+Note: the `SHOW` statement for session settings in unrelated to the
+other `SHOW` statements:
+[`SHOW CREATE TABLE`](show-create-table.html),
+[`SHOW CREATE VIEW`](show-create-view.html),
+[`SHOW USERS`](show-users.html),
+[`SHOW DATABASES`](show-databases.html),
+[`SHOW COLUMNS`](show-columns.html),
+[`SHOW GRANTS`](show-grants.html), [`SHOW INDEX`](show-index.html) and
+[`SHOW CONSTRAINTS`](show-constraints.html).
+
+## Supported variables
+
+|Variable | Can be modified with [`SET`](set-vars.html)? | Description | Initial value |
```

Comment:
Same comment as with `SHOW` in `set-vars.md`, where we need an explanation of `SET` and this column might be better suited for the final column.